qemu/tests/virtio_console.py

```diff
@@ -570,12 +571,154 @@ def test_loopback():
             logging.error(msg)
             test.fail(msg)
 
+    def get_virtio_port_by_name(vm, name):
+        """
+        Get virtio port object by name in VM.
+        :param vm: vm object in this test
+        :param name: name of the port
+        """
+        for device in vm.devices:
+            if isinstance(device, qdevices.QDevice):
+                if device.get_param("name") == name:
+                    return device
+        return None
```

Comment:
Is this part duplicate with https://github.com/autotest/tp-qemu/pull/1037 ?

Reply:
Yes, will remove it from this patch, thanks!